### ISSUE TITLE
Smaller buttons

### ISSUE THREAD

**Issue by fuzzy76** [heiseonline/shariff#184]
Since #146 was closed, I'm re-issuing as a feature request. More specifically, I'd like "textless" buttons like the [Drupal module shows](https://www.drupal.org/project/shariff).

**Comment by LoyalBen** [comment]
+1 
Yes, smaller buttons would be great!

**Comment by JoergLueg** [comment]
Hello

How can you display the buttons without the text, so only on icon size.
I use the Java version. Can someone help me.

Greetings Jörg

**Comment by koelling** [comment]
FWIW, I have used the following CSS to remove the text and make the buttons smaller. Didn't notice any problems so far.

```
.shariff .orientation-horizontal li {
	min-width: auto;
	max-width: 60px;
}

.shariff .orientation-horizontal li .share_text {
	display: none;
}
```

**Comment by richard67** [comment]
Well I think it would be a good feature, but not as a theme like it was suggested in issue #284 .

It should be a new, separate option, like e.g. data-button-size or so. It could be used with any of the themes standard, grey or white then.

Maybe I find time on weekend to implement that and make a pull request here.

**Comment by richard67** [comment]
I think I need more time. Maybe someone else wants to do that?

**Comment by richard67** [comment]
Guys, I made a pull request (PR) for having small icons with or without the share counters. Please test PR #309 and report the results and your opinion and possible suggestions there with comments. Testing instructions and links to texting packages you can find in the description of that PR.

**Comment by richard67** [comment]
I see my solution was released with version 3.0.0 now. Can you check if it fulfills your requirements, and if yes, close this issue?

**Comment by fuzzy76** [comment]
If that was directed at me, feel free to close. I don't have time to upgrade the site I'm using Shariff on for a while, so I shouldn't hold up this issue. :)
